be/src/util/runtime_profile.h

```diff
@@ -613,9 +623,18 @@ class RuntimeProfile {
     // of the total time in the entire profile tree.
     double _local_time_percent;
 
-    // update a subtree of profiles from nodes, rooted at *idx.
+    // Protects _version
+    mutable std::mutex _version_lock;
+    // The version of this profile. It is used to prevent updating this profile
```

Comment:
Why not use atomic<int64_t>?

Reply:
need to update `_version` according some conditions